agagd/templates/static_pages/information.html

```diff
@@ -26,7 +26,25 @@ <h2>Additional Information</h2>
     <li>Answers to the <a href="https://www.usgo.org/files/pdf/ratingquestions.pdf">most commonly asked questions about the rating system</a> (May 1992)</li>
     <li>A more detailed <a href="https://www.usgo.org/files/pdf/AGARatings-Math.pdf">mathematical description</a> of the ratings system (April 2010)</li>
     <li><a href="https://www.usgo.org/files/pdf/bayrate.zip">Source code</a> for the AGA rating algorithm</li>
-    <li>Tournament Directors: <a href="https://usgo.org/mm/tdlista.txt">TDListA</a>  <a href="https://usgo.org/mm/tdlistn.txt">TDListN</a></li>
+    <li>Tournament Directors: <br>
+        <table>
+            <tr>
+                <th>TD List Name</th>
+                <th><a href="https://usgo.org/mm/tdlista.txt">TDListA</a></th>
+                <th><a href="https://usgo.org/mm/tdlistb.txt">TDListB</a></th>
+                <th><a href="https://usgo.org/mm/tdlistn.txt">TDListN</a></th>
+            </tr><tr>
+                <th>Delimiter</th>
+                <th>Tab</th>
+                <th>Tab</th>
+                <th>Tab</th>
+            </tr><tr>
+                <th>Columns displayed</th>
+                <th>MememberName AGAID MememberType Rating RenewalDue ChapterCode State Sigma RatingUpdated</th>
+                <th>MememberName AGAID MememberType Rating RenewalDue ChapterName State Sigma RatingUpdated</th>
+                <th>MememberName AGAID MememberType Rating RenewalDue ChapterCode State || Fixed Width</th>
+            </tr>
+        </table>  </li>
```

Comment:
In my opinion, I think these should be a little clearer that we are using tab delimitation for our TDLists. 

I think we could do something about the presentation of the table and it should be a different section. Under the bullet list may not be clear to some.
![image](https://user-images.githubusercontent.com/5054653/103465428-2ade3600-4cf0-11eb-818a-d5789ec6d939.png)

Reply:
Youre right, I was writing these one way and they need to be the other way. I will resubmit this.